classes/options-general.php

```diff
@@ -216,36 +216,36 @@ function print_page() {
 
 											if ( ! url_has_ftps_for_item( $simplemap_ps ) ) : ?>
 
-												<h4><?php printf( __( '<strong>SimpleMap Premium Support Benefits - $30/year/domain</strong>', 'simplemap' ), esc_attr( site_url() ) ); ?></h4>
+												<h4><?php printf( __( '<strong>SimpleMap Premium Support Benefits - $30/year/domain</strong>', 'SimpleMap' ), esc_attr( site_url() ) ); ?></h4>
 												<p>
-													<?php //printf( __( 'SimpleMap now offers a premium support package for the low cost of %s per year per domain.', 'simplemap' ), '$30.00 USD' ); ?>
+													<?php //printf( __( 'SimpleMap now offers a premium support package for the low cost of %s per year per domain.', 'SimpleMap' ), '$30.00 USD' ); ?>
 												</p>
 												<p>
-													<?php _e( '<em>By signing up for SimpleMap premium support, you help to ensure future enhancements to this excellent project as well as the following benefits:</em>', 'simplemap' ); ?>
+													<?php _e( '<em>By signing up for SimpleMap premium support, you help to ensure future enhancements to this excellent project as well as the following benefits:</em>', 'SimpleMap' ); ?>
 												</p>
 
 												<ul style='margin-left:25px;list-style-type:disc'>
-													<li><?php _e( 'Around the clock access to our extensive knowledge base and support forum from within your WordPress dashboard', 'simplemap' ); ?></li>
-													<li><?php _e( 'Professional and timely response times to all your questions from the SimpleMap team', 'simplemap' ); ?></li>
-													<li><?php _e( 'A 10% discount for any custom functionality you request from the SimpleMap developers', 'simplemap' ); ?></li>
-													<li><?php _e( 'Advance access to new features integrated into the auto upgrade functionality of WordPress', 'simplemap' ); ?></li>
+													<li><?php _e( 'Around the clock access to our extensive knowledge base and support forum from within your WordPress dashboard', 'SimpleMap' ); ?></li>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 202 characters

<!-- Comment by [SideCI](https://sideci.com) -->